### PR TITLE
Configuration documentation updates

### PR DESCRIPTION
**Pull request by tobiasake**
### Description of the Change
Configuration updates about the two new ways of executing RemRem-Generate service with java and maven commands.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: serviceUsage.html [hunk 1]
```diff
@@ -66,9 +66,24 @@ <h2>
         <pre>set JAVA_OPTS="-Djava.ext.dirs=/path/to/jars/" in catalina.sh</pre>
         <p><strong>NOTE:</strong> in the above example, protocol jar file must be present inside "/path/to/jars/" folder.</p>
         <p><strong>NOTE:</strong> <code>-Djava.ext.dirs</code> is no longer working in some JAVA8 versions and in JAVA9.
-            So users should create a wrapper project to include both Generate/Publish and their protocol in
+            So users should create a wrapper project to includse both Generate/Publish and their protocol in
             or place the protocol library in the folder for external dependencies of their JVM installation.
         </p>
+
```
Comment (evasiba-ericsson) [inline serviceUsage.html]: The section for running inside tomcat should get its own header and should be moved after the the section for running without tomcat.

File: serviceUsage.html [hunk 2]
```diff
@@ -66,9 +66,24 @@ <h2>
         <pre>set JAVA_OPTS="-Djava.ext.dirs=/path/to/jars/" in catalina.sh</pre>
         <p><strong>NOTE:</strong> in the above example, protocol jar file must be present inside "/path/to/jars/" folder.</p>
         <p><strong>NOTE:</strong> <code>-Djava.ext.dirs</code> is no longer working in some JAVA8 versions and in JAVA9.
-            So users should create a wrapper project to include both Generate/Publish and their protocol in
+            So users should create a wrapper project to includse both Generate/Publish and their protocol in
```
[the remark this answers is not in the document]
Comment (tobiasake) [reply serviceUsage.html]: Fixed.

File: serviceUsage.html (hunk 1 above)
Comment (tobiasake) [reply serviceUsage.html]: Fixed.

File: serviceUsage.html [hunk 3]
```diff
@@ -55,8 +55,24 @@ <h2>
...
+        <pre>java -jar service/target/generate-service-x.x.x.war --spring.config.location=path/to/applicattion.propeties</pre>
+
+        <p>Second option to execute RemRem-Generate is to execute maven commands from the source code root folder, example:</p>
+        <pre>mvn spring-boot:run -Dspring-boot.run.arguments=--properties.parameter1=value,--properties.parameter2=value,--properties.parameter3=value,--properties.parameter4=value ......</pre>
+        <p>When using maven command, Spring properties can also be changed by editing service/src/main/resources/application.properties file.</p>
+
+
+        <h4>Running and configuring in Tomcat</h4>
+        <p>Eiffel REMReM Service <strong>generate-service.war</strong> file should be deployed in Tomcat Server.
+            For doing this, generate-service.war file should deployed in directory: <em>tomcat/webapps</em>.
```
Comment (Christoffer-Cortes) [inline serviceUsage.html]: I don't know how strict we are with grammar but here goes. 'For doing this' should be 'To do this' and there is also a 'be' missing here 'file should be deployed'

**Pull request merged**